.template-lintrc.js

```diff
@@ -1,5 +1,8 @@
 'use strict';
 
 module.exports = {
-  extends: 'octane'
+  extends: 'octane',
+  rules: {
+    'require-button-type': false
```

Comment:
Why? 🤔 😜

Reply:
Why does a button need a type? :/

Reply:
https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rule/require-button-type.md
Most buttons aren't "submit" buttons :)

Reply:
but buttons outside of a form with type button feels redundant. why is default submit. :-\

Reply:
Ha blame the web